fairroot/base/steer/FairRootManager.h

```diff
@@ -67,12 +67,8 @@ class FairRootManager : public TObject
     0 : Branch does not exist   */
     Int_t CheckBranch(const char* BrName);
 
-    void CloseSink()
-    {
-        if (fSink) {
-            fSink->Close();
-        }
-    }
+    // TODO: Redundant API? fSink is exposed and users can directly call FairSink::Close
+    void CloseSink() { fSink->Close(); }
```

Comment:
Please keep the `if (fSink)`. We can't guarantee that it is non-nullptr.

(I think, we even have some valid use cases for this one.)